scripts/test_pip_install.sh

```diff
@@ -106,7 +107,7 @@ python3 -m clouddq ALL configs \
 
 if [[ -f "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
     # test clouddq with exported service account key
-    python3 -m clouddq ALL configs \
+    python3 -m clouddq T1_DQ_1_VALUE_NOT_NULL,T2_DQ_1_EMAIL,T3_DQ_1_EMAIL_DUPLICATE configs \
```

Comment:
Is this change related to the throttling issue? If not, why would you need to explicitly set the bindings to be run?

Reply:
Yes because ALL includes rule_bindings with entity_uris which will trigger the dataplex metadata API calls, leading to run failures.

Reply:
ok, gotcha! makes sense